src/include/migraphx/op/pointwise.hpp

```diff
@@ -45,14 +45,15 @@ struct pointwise
         {
             MIGRAPHX_THROW("should have one submodule.");
         }
-        auto* pm    = mods.front();
+        auto* pm = mods.front();
+        if(pm->get_output_shapes().size() != 1)
+            MIGRAPHX_THROW("submodule should have only one output.");
+        if(inputs.empty())
+            MIGRAPHX_THROW("submodule should have atleast one input");
```

Comment:
This should probably say `pointwise should have at least one input`.

Reply:
Done